sdcm/cluster.py

```diff
@@ -272,9 +272,16 @@ def get_startup_script(cls):
         post_boot_script = '#!/bin/bash'
         if cls.RSYSLOG_ADDRESS:
             post_boot_script += dedent('''
-                   sudo echo 'action(type="omfwd" Target="{0}" Port="{1}" Protocol="tcp")'>> /etc/rsyslog.conf
-                   sudo systemctl restart rsyslog
-                   '''.format(*cls.RSYSLOG_ADDRESS))
+                sudo echo 'action(type="omfwd" Target="{0}" Port="{1}" Protocol="tcp")'>> /etc/rsyslog.conf
+                sudo systemctl restart rsyslog
+            '''.format(*cls.RSYSLOG_ADDRESS))
+        post_boot_script += dedent('''
+               sudo sed -i 's/#MaxSessions \(.*\)$/MaxSessions 1000/' /etc/ssh/sshd_config
+               sudo systemctl restart sshd
```

Comment:
did you tests this? I don't how sshd would react to restart during execution of the script using itself :)

Reply:
Yes, tested it, and the thing here is that this script is being executed in the instance initiation.
In any case, you can restart sshd service, as it won't allow new connections (while restarting) but it won't disconnect you from an existing one.